task2/src/components/AppButton/AppButton.jsx

```diff
@@ -0,0 +1,9 @@
+import styled from "styled-components";
+
+const AppButtonStyled = styled.button`
```

Comment:
do not use this approach of  naming `*Styled` 

it's enough to know that it was imported from `Button.styles.jsx` file

Reply:
> do not use this approach of naming `*Styled`
> 
> it's enough to know that it was imported from `Button.styles.jsx` file

Got it!